iModelCore/BeSQLite/PublicAPI/BeSQLite/BeSQLite.h

```diff
@@ -2487,6 +2499,7 @@ struct EXPORT_VTABLE_ATTRIBUTE Db : NonCopyableClass
         ProfileUpgradeOptions m_profileUpgradeOptions = ProfileUpgradeOptions::None;
         bool m_rawSQLite = false;
         bool m_fromContainer = false;
+        bool m_allowDataTransformDuringSchemaUpdate = false;

```

Comment:
this should be m_schemaLockHeld. If we could test for that fact in C++ it wouldn't be necessary, but this isn't a programmer choice, it's a fact that our TS layer has to set correctly. In other words you don't turn on this flag, you acquire the schema lock and *that* turns on this flag.

Reply:
I have changed the name in c++ layer.